packages/vars/src/typography.css

```diff
@@ -47,52 +47,62 @@
   letter-spacing: -0.5px;
   font-weight: var(--font-weight-bold);
   font-family: var(--font-family-system);
+  line-height: 46px;
 }
 
 @define-mixin typography-heading-2_mobile {
   @mixin typography-heading-2;
   font-size: 28px;
+  line-height: 40px;
 }
 
 @define-mixin typography-heading-3 {
   font-size: 28px;
   letter-spacing: -0.5px;
   font-weight: var(--font-weight-bold);
   font-family: var(--font-family-system);
+  line-height: 40px;
 }
 
 @define-mixin typography-heading-3_mobile {
   @mixin typography-heading-3;
   font-size: 24px;
+  line-height: 35px;
 }
 
 @define-mixin typography-heading-4 {
   font-size: 24px;
   letter-spacing: -0.5px;
   font-weight: var(--font-weight-bold);
   font-family: var(--font-family-system);
+  line-height: 35px;
```

Comment:
BTW, all of your changes for line-height, it's not correct because the designers team decided on the set setting for each typography variant.
You can find it here: https://www.figma.com/file/rrvkvQEoVTOHa7MzyKSaoz/OTP-Global-Design-System?node-id=1369%3A39089

Reply:
Hey @derek-heycar, these values are coming from the link you shared. So they should be okay.

Reply:
<img width="665" alt="image" src="https://user-images.githubusercontent.com/89923576/187675974-26a2f0be-62da-4961-9106-985064937276.png">

Reply:
we actually didn't see those information and took the values from `inspect`. But anyway thanks for pointing this out. <img width="578" alt="Screenshot 2022-08-31 at 14 15 27" src="https://user-images.githubusercontent.com/45823795/187681821-3aa5db97-d7a3-489c-8580-1d60e8496ef1.png">